### ISSUE TITLE
Play audio cue when a command exits with non-zero code

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#174368]
@jooyoungseo conveyed how useful audio cues are. With shell integration, we know when a command fails. We should play a failure sound when a command exits with a non-zero exit code.

We already play a sound when a notebook cell or task fails, so can use that.
![The Settings UI is open to the audio cue notebook cell and task failed settings](https://user-images.githubusercontent.com/29464607/218793387-e1808f33-7395-416d-a7d6-0fb158e98a75.png)
